### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by neutric**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -36,7 +36,7 @@ WHY IS PAR 2.0 better than PAR 1.0?
    repair. It is therefore not necessary to create as much recovery
    data in the first place to achieve the same level of protection.
 
- * You can protect up to 32768 files rather than the 256 that PAR 1.0
+ * You can protect up to 32,768 files rather than the 256 that PAR 1.0
```
Comment (BlackIkeEagle) [inline README]: I think this change is not usefull. There are so many 'notation' thingies around the world I rather have all numbers together than in some specific localized format.

Comment (neutric) [inline README]: I absolutely don't insist on it and 5 digits are easy enough to discern without thousands separators. Can you merge selectively or would you like me to undo this change in another commit?

**Comment by BlackIkeEagle** [comment]
I've created a new PR with the minor update #75

**Comment by BlackIkeEagle** [comment]
Thanks for these improvements @neutric
